src/server/api/controllers/users.controller.js

```diff
@@ -3,25 +3,24 @@ Can be deleted as soon as the first real controller is added. */
 
 const knex = require('../../config/db');
 const Error = require('../lib/utils/http-error');
-const moment = require('moment-timezone');
 
 const getUsers = async () => {
   try {
-    return await knex('users').select('users.id', 'users.title');
+    return await knex('users').select('*');
   } catch (error) {
     return error.message;
   }
 };
 
 const getUsersById = async (id) => {
   try {
-    const users = await knex('users')
-      .select('users.id as id', 'title')
+    const user = await knex('users')
+      .select('*')
       .where({ id });
-    if (users.length === 0) {
+    if (user.length === 0) {
       throw new Error(`incorrect entry with the id of ${id}`, 404);
     }
-    return users;
+    return user;
```

Comment:
This will return an array of users, maybe users will be more correct?

Suggestion: Change to
```
return users[0];
```

Reply:
each user has unique id. We´re getting one user with a specific id ` .where({ id });`. Therefore, no need for [0], the function will return an array with only one object of a selected user with all the info for her/him from the database :)

Reply:
Maybe the function name confused you? I left it as plural (users) because it is a reusable function and we can reuse it to select different users. But with the function we select one user at a time.

Reply:
Oh yeah, you're right, I think it seems good :)